addons/l10n_pl/data/account_tax_report_data.xml

```diff
@@ -166,22 +166,22 @@
     <record id="account_tax_report_line_do_przeniesienia" model="account.tax.report.line">
         <field name="name">Podatek - Do przeniesienia</field>
         <field name="sequence" eval="3"/>
-        <field name="formula">PLTAXC_04_16 + PLTAXC_05_18 + PLTAXC_06_20 + PLTAXC_09_24 + PLTAXC_10_26 + PLTAXC_11_28 + PLTAXC_12_32 + PLTAXD_02_42</field>
+        <field name="formula">PLTAXC_04_16 + PLTAXC_05_18 + PLTAXC_06_20 + PLTAXC_09_24 + PLTAXC_10_26 + PLTAXC_11_28 + PLTAXC_12_32 - PLTAXD_02_42</field>
         <field name="country_id" ref="base.pl"/>
     </record>
 
     <record id="account_tax_report_line_nad_naleznym" model="account.tax.report.line">
         <field name="name">Podatek - Nadwyżka naliczonego nad należnym</field>
         <field name="sequence" eval="1"/>
-        <field name="formula">PLTAXC_04_16 + PLTAXC_05_18 + PLTAXC_06_20 + PLTAXC_09_24 + PLTAXC_10_26 + PLTAXC_11_28 + PLTAXC_12_32 + PLTAXD_02_42</field>
+        <field name="formula">PLTAXC_04_16 + PLTAXC_05_18 + PLTAXC_06_20 + PLTAXC_09_24 + PLTAXC_10_26 + PLTAXC_11_28 + PLTAXC_12_32 - PLTAXD_02_42</field>
         <field name="parent_id" ref="account_tax_report_line_do_przeniesienia"/>
         <field name="country_id" ref="base.pl"/>
     </record>
 
     <record id="account_tax_report_line_do_US" model="account.tax.report.line">
         <field name="name">Podatek - Do wpłaty do US</field>
         <field name="sequence" eval="1"/>
-        <field name="formula">PLTAXC_04_16 + PLTAXC_05_18 + PLTAXC_06_20 + PLTAXC_09_24 + PLTAXC_10_26 + PLTAXC_11_28 + PLTAXC_12_32 + PLTAXD_02_42</field>
+        <field name="formula">PLTAXC_04_16 + PLTAXC_05_18 + PLTAXC_06_20 + PLTAXC_09_24 + PLTAXC_10_26 + PLTAXC_11_28 + PLTAXC_12_32 - PLTAXD_02_42</field>
```

Comment:
Fine for 13.0, but in 14.0 please add a code on `account_tax_report_line_podatek_razem_c` and `account_tax_report_line_podatek_razem_d` and use them here
See the description of the commit for more info https://github.com/odoo/enterprise/commit/5d7ed69e423d6e6814a5a185279afd3b634c531b

Reply:
Ok, it's just that I wanted the smallest modification so it changes the least for this fix. I have a bigger revamp planned for the Polish localization with https://github.com/odoo/odoo/pull/87035, where the tax report will be completely changed.

Reply:
Ok, but you are still using a raw declaration
https://github.com/odoo/odoo/pull/87035/commits/6eb8985e719551e143c03b8ba20d276f12a179d9#diff-4192d27fc406f02e4a9f906b394e7f1624d76d52e6a40782d8915d5af23b625cR188

Reply:
I just tend to prefer doing the explicit sum of children on this line (I don't know if it is better or not), but the intermediary codes are used when I can: https://github.com/odoo/odoo/commit/6eb8985e719551e143c03b8ba20d276f12a179d9#diff-4192d27fc406f02e4a9f906b394e7f1624d76d52e6a40782d8915d5af23b625cR180

Reply:
> I just tend to prefer doing the explicit sum of children on this line (I don't know if it is better or not)

Why would it be better? Isn't it just adding complexity to understand where the different values come from?

> but the intermediary codes are used when I can

Why can't you in this case?